src/components/Footer/Footer.jsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
-import { Typography, Link } from '@mui/material';
+import { Link } from '@mui/material';
 import { FooterContainer, FooterLinks } from './Footer.styled.js';
 import { Link as RouterLink } from 'react-router-dom';
 import { ROUTES as ROUTER_PATHS } from '../../constants/routerPaths.js';
+import SharedTypography from '../shared/Text/SharedText.jsx';
 
 function Footer() {
   return (
     <FooterContainer component="footer">
-      <Typography variant="body2" color="textSecondary" sx={{ mb: 1 }}>
+      <SharedTypography variant="body2" sx={{ mb: 1 }}>
         © {new Date().getFullYear()} SecondHand Store
```

Comment:
Your SharedTypography accept style object, use it instead of inline style using sx